miso-web/src/main/webapp/scripts/labeldesign.js

```diff
@@ -0,0 +1,606 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
```

Comment:
Is this still an accurate contact for new files?

Reply:
Good question. Dillan, what are we doing for new files?

Reply:
Mostly not including it at all

Reply:
I like this solution best.